packages/ui/v2/core/WizardForm.tsx

```diff
@@ -52,8 +52,8 @@ function WizardForm<T extends DefaultStep>(props: { href: string; steps: T[]; co
               loading={currentStep.isLoading}
               type="submit"
               color="primary"
-              form={`setup-step-${step}`}
```

Comment:
"setup" felt really tailored to the usage in `/auth/setup`. Changed it to `wizard` so it's a reusable id in other forms as well.